modules/ROOT/pages/publish-apis.adoc

```diff
@@ -4,70 +4,114 @@ include::_attributes.adoc[]
+Open the ACM administrator interface and select the *Manage APIs* tab at the top left.
 
-Each of your Anypoint organization's APIs are shown in one of the lists.
+On this tab you can manage what APIs from Exchange are presented in your community and their metadata and visibility.
 
-Each item in the list *APIs in this Community* shows an icon, name, visibility icon, description, and an *Edit* button.
+The section *APIs in this Community* shows the API versions that have been added to the community.
 
-Public APIs show the visibility icon of a blue eye. APIs visible only to registered users show a black eye. An API in preview mode is only visible to the community administrator who added it and shows a black eye with a line through it.
+The section *APIs in Exchange* shows the API versions that exist in your Exchange organization and have not been added to the community.
 
-Each item in the list *APIs in Exchange* shows an icon, name, description, and *Add to Community* button.
+To add an API version from Exchange to the Community, click the *Add to Community* button to the right of the API version name.
 
-The lists are divided into pages and show page number links at the bottom. To change the number of items shown, use the menu icon with three horizontal lines in the top right corner.
+The API version is now listed in the *APIs in this Community* section. *Visibility* is set to *Admin Only*, the API version has no icon, and the *Description* is blank.
 
-You can search the APIs by name with the *APIs Search* bar at the top.
+== Set API Information
 
-== Add an API to your community
+Click the *Edit* button to the right of the API version name.
 
-To add an API from Exchange to your community, list your community's APIs, find the API in the list *APIs in Exchange*, and click the button *Add to Community*.
+Keep the current *API Name*.
 
-The API is removed from the list *APIs in Exchange* and appears in the list *APIs in this Community* in preview mode, so it is only visible to the community administrator who added it. To complete additional API information (including its icon, description, and link to a community page with business details), and change its visibility, edit the API.
+Click *Replace Icon* to set the API version icon.
 
-== Edit an API
+Add a *Description* for the API version.
 
-To edit an API, list your community's APIs, find the API in the list *APIs in this Community*, and click the button *Edit*.
+For now, leave the *Visibility* as *Admin Only* while you preview the API version.
 
-The editing screen shows:
+By default each API version on your community's page has a link labeled "API Details" to a page showing information about the API. To preserve this default, set *Marketing Page* to *None*. To link to a different CMS content page in your community such as a marketing page, set the *Marketing Page* option to *CMS/Custom page* and fill in the URL of the CMS page.
```

Comment:
By default, each API version on your community's page has a link (labeled "API Details") to a page...

Reply:
That is better. Fixed.